Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationDataSource.java

```diff
@@ -173,4 +220,52 @@ public Long getDataSourceObjectID() {
     public String getName() {
         return name;
     }
+    
+    /**
+     * @return the MD5 hash value
+     */
+    String getMd5() {
+        return md5Hash;
+    }
+    
+    /**
+     * Set the MD5 hash value.
```

Comment:
We should add comments that this does not actually save it to the DB and that you need to call some other method for them to be persisted.

Reply:
I'm not sure it is worth having a separate method for this. Let's discuss via email.